wp-content/themes/blank/template-parts/panel-section.php

```diff
@@ -0,0 +1,41 @@
+<?php if( have_rows('panel_section_repeater') ): ?>
+
+<div class="panel-section">
+    <div class="container">
+<?php while( have_rows('panel_section_repeater') ): the_row(); 
+
+    // vars
+    $link = get_sub_field('panel_link');
+    $title = get_sub_field('panel_title');
+    $subTitle = get_sub_field('panel_sub_title');
+    $image = get_sub_field('panel_image');
+
+    ?>
+
+    <div class="panel">
+
+    <?php if( $link ): 
+        $link_url = $link['url'];
+        $link_title = $link['title'];
+        $link_target = $link['target'] ? $link['target'] : '_self';
+        ?>
+        <a class="button" href="<?php echo esc_url( $link_url ); ?>" target="<?php echo esc_attr( $link_target ); ?>"><?php echo esc_html( $link_title ); ?>
+    <?php endif; ?>
+
+            <h3><?php echo $subTitle; ?></h3>
+            <h2><?php echo $title; ?></h2>
```

Comment:
if consuming once, why add to a variable?
```suggestion
            <h3><?php get_sub_field('panel_sub_title'); ?></h3>
            <h2><?php get_sub_field('panel_title'); ?></h2>
```

Reply:
There was a talk about having it more than once so I left it there just in case.